database_cleanup/models/purge_models.py

```diff
@@ -87,11 +87,15 @@ def purge(self):
                     pass
                 except AttributeError:
                     pass
+            if line.name not in self.env:
```

Comment:
do we need this now that the test runs at the end of everything? The problem is that for the actual use case of the module, this will happen very often, and reloading the registry for every unknown model will take very long. The alternative would be to check for the testing flag and do it only then if it's still necessary for the test